docs/root/_include/ssl_stats.rst

```diff
@@ -18,3 +18,4 @@
    curves.<curve>, Counter, Total successful TLS connections that used ECDHE curve <curve>
    sigalgs.<sigalg>, Counter, Total successful TLS connections that used signature algorithm <sigalg>
    versions.<version>, Counter, Total successful TLS connections that used protocol version <version>
+   was_key_usage_invalid, Counter, Total successful TLS connections that used protocol version has invalid RSA key usage
```

Comment:
```suggestion
   was_key_usage_invalid, Counter, Total successful TLS connections that used an invalid RSA key usage
```

Can you link to some docs (either other envoy docs, or boringssl docs) that describe what this means?